src/main/java/io/r2dbc/pool/ConnectionPool.java

```diff
@@ -129,11 +129,9 @@ public ConnectionPool(ConnectionPoolConfiguration configuration) {
         if (maxSize == -1) {
             builder.sizeUnbounded();
         } else {
-            builder.sizeMax(maxSize);
+            builder.sizeBetween(initialSize, maxSize);
```

Comment:
We still need to consider the minimum size. Throughout the course of https://github.com/reactor/reactor-pool/pull/63, we decided to remove `sizeMin`/`sizeMax` methods and retain`sizeBetween`. Care to update your pull request to also consider the initial size?